three/index.d.ts

```diff
@@ -261,7 +261,7 @@ declare namespace THREE {
         reset(): AnimationAction;
         isRunning(): boolean;
         startAt(time: number): AnimationAction;
-        setLoop(mode: boolean, repetitions: number): AnimationAction;
+        setLoop(mode: AnimationActionLoopStyles, repetitions?: number): AnimationAction;
```

Comment:
Hmm, I think you're right. I've been passing in undefined and it's worked, but it looks like it only works by accident (specifically [this check](https://github.com/mrdoob/three.js/blob/dev/src/animation/AnimationAction.js#L540) fails if pending is undefined).